roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+"""
+  Interface to OpenShift oc command
+"""
+#
+#   Copyright 2015 Red Hat Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import os
+import shlex
+import atexit
+import shutil
+import string
+import random
+import subprocess
+import yaml
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+# pylint: disable=bare-except
+def cleanup_file(inc_file):
+    """ clean up """
+    try:
+        os.unlink(inc_file)
+    except:
+        pass
+
+
+class OCUtil(object):
```

Comment:
It looks like `openshift_tools` is just a set of Python scripts unrelated to Ansible...
Moving the canonical source of the file here doesn't make sense, they shouldn't try to reuse an "Ansible module" (similar but != "Python module") from this repo and bring in all the versioning and bookkeeping headache.

Makes even less sense to go that far to grab code instead of:
1. Keep it simple and use the stdlib unless it pays off to bring in a dependency
2. Use code from the openshift-ansible repo, see: https://github.com/openshift/openshift-ansible/blob/b47708f36d0e45688d4444ca22f6a62d0c5de2a6/roles/lib_openshift/src/lib/base.py#L251

Option no. 1 is highly preferable if all you need is to make simple `oc` calls:

https://docs.python.org/2/library/subprocess.html#using-the-subprocess-module